ldap/ldif/template-dse-minimal.ldif.in

```diff
@@ -208,6 +209,7 @@ nsslapd-pluginDescription: PBKDF2
 dn: cn=PBKDF2-SHA1,cn=Password Storage Schemes,cn=plugins,cn=config
 objectclass: top
 objectclass: nsSlapdPlugin
+objectClass: pwdPBKDF2PluginConfig
```

Comment:
I see that you are modifying the dse.ldif. 
  What will happen on upgraded instances ?

Reply:
For the upgrade, I have a logic in CLI tools.
It will add the `pwdPBKDF2PluginConfig` objectClass if it's not present when the admin tries to add `nsslapd-pwdPBKDF2NumIterations`.
See `src/lib389/lib389/cli_conf/plugins/pwstorage.py`

Otherwise, the PBKDF2-* Rust logic uses a default value. (And CLI would return the default if the attr is not present)